transport/src/main/java/io/netty/channel/ChannelHandler.java

```diff
@@ -277,12 +277,11 @@ default void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws
      *
```

Comment:
This is the most interesting part of this PR in terms of API changes @netty/netty-5